src/shared/components/Header/interfaces.ts

```diff
@@ -0,0 +1,4 @@
+export interface MenuListItemI {
```

Comment:
also, I would suggest to call it `IMenuListItem` - to be more clear that it's an interface

also, why do we need interface here, not just a type?

Reply:
eslint displays an error